src/main/kotlin/io/hsar/wh40k/combatsimulator/random/AverageDice.kt

```diff
@@ -9,4 +9,8 @@ object AverageDice {
     private val roller = DiceRoller(AverageNumberGenerator())
 
     fun roll(diceString: String): Int = parser.parse(diceString, roller).totalRoll
+
+    fun averageDegrees(skillTarget: Int) {
+
```

Comment:
Is this... correct?

Reply:
Its an unused method that I must've forgotten to remove. It is now removed.